ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5164,3 +5167,110 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ medium_request_size_limit }}"
 
+  - name: createObjectCategory
+    uris: "{{ object_category_prefix }}/v1/create"
+    upstream_url: "{{ taxonomy_service_url }}/object/category/v4/create"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'frameworkCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: readObjectCategory
+    uris: "{{ object_category_prefix }}/v1/read"
+    upstream_url: "{{ taxonomy_service_url }}/object/category/v4/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'frameworkCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: updateObjectCategory
+    uris: "{{ object_category_prefix }}/v1/update"
+    upstream_url: "{{ taxonomy_service_url }}/object/category/v4/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'frameworkCreate'
```

Comment:
change the ACL to update group and not create @krgauraw .

Reply:
done